test_base58.py

```diff
@@ -66,6 +71,12 @@ def test_check_str():
     assert_that(back, equal_to(b'hello world'))
 
 
+def test_autofix_check_str():
```

Comment:
Could you also add a test with a custom alphabet where the autofix is not applied

Reply:
fix in 54f46056283a2c5004a8bca8f89d9889d06b3161